nbs/introduction.ipynb

```diff
@@ -89,147 +89,147 @@
      "name": "stderr",
```

Comment:
<p>I dont think this "empty table" will persist in later versions, and are queries even saved in the DB?</p><p> </p><p>What the users should know here is that we save the facts in the database, the rules' logic is saved seperately and only evaluated lazily (on demand).</p><p> </p><p>One a query is issued, the engines uses the rules to derive all possible solutions from the existing facts that would satisfy the query.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/spanner_workbench/pull/134/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>